### ISSUE TITLE
Document CAIP2 for all namespaces supported by The Graph

### ISSUE THREAD

**Issue by DenisCarriere** [pinax-network/supported-networks#17]
- [ ] Document CAIP2 for all namespaces supported by The Graph
  - [ ] Near
- [ ] Provide bash script or query method to retrieve Chain ID to be used in CAIP2
- [ ] Include this documentation in `./docs/caip2.md` document

**Comment by YaroShkvorets** [comment]
What does it involve? Submitting PR to https://github.com/ChainAgnostic/CAIPs/ for chains that are not defined there, like Near?

**Comment by matthewdarwin** [comment]
Yes, PRs can be submitted.  I watched the process of antelope chains being added around 1 year back.

**Comment by YaroShkvorets** [comment]
Feels kinda weird to define a standard for a chain I never even transacted in

**Comment by matthewdarwin** [comment]
Probably SF team has contacts at near to get them to work on this.
